policyengine_us/parameters/gov/states/ca/tax/income/credits/foster_youth/phase_out/amount.yaml

```diff
@@ -0,0 +1,15 @@
+description: California reduces the foster youth tax credit by this amount for each increment.
+values:
+  2022-01-01: 20
+metadata:
+  unit: currency-USD
+  period: year
+  label: California foster youth tax credit phase out amount
+  reference:
+  # The foster youth tax credit was implemented  in 2022
+    - title: California Foster Youth Tax Credit
+      href: https://www.ftb.ca.gov/file/personal/credits/foster-youth-tax-credit.html
+    - title: 2022 California Earned Income Tax Credit Form
+      href: https://www.ftb.ca.gov/forms/2022/2022-3514.pdf#page=4
+    - title: California Revenue and Taxation Code (R&TC) Section 17052.2(a)
```

Comment:
please be precise with the references: 
this should be:
Section 17052.2 (a) (2) (C) (i)